AstraEngine.Core/Entity.cs

```diff
@@ -167,4 +231,14 @@ public void AddChild(Entity player)
     {
         player.Parent = this;
     }
+
+    /// <summary>
+    /// Removes the specified Entity as a child of this component
+    /// </summary>
+    /// <param name="player"></param>
+    public void RemoveChild(Entity player)
+    {
+        player.Parent = null;

```

Comment:
* What happens when the entity is not a child of this Entity?
* What happens if the entity was already a child of another Entity?